templates/copy-template/generate-task.yaml

```diff
@@ -12,21 +12,27 @@
     - name: "Generate {{ task_name }} task"
       template:
         src: "{{ manifest_templates_dir }}/{{ task_name }}.yaml"
-        dest: "{{ manifests_output_dir_tmp }}/{{ task_name }}.yaml"
+        dest: "{{ manifests_output_dir }}/{{ task_name }}.yaml"
         mode: "{{ default_file_mode }}"
-    - name: Generate roles
-      include_tasks: "{{ repo_dir }}/scripts/ansible/generate-roles.yaml"
-      with_items:
-        - { role_type: ClusterRole, prefix: zz- }
-      vars:
-        role_output_dir: "{{ manifests_output_dir_tmp }}"
     - name: Prepare examples dist directory
       file:
         path: "{{ item }}"
         state: directory
       with_items:
         - "{{ examples_output_dir }}"
         - "{{ examples_taskruns_output_dir }}"
+    - name: Prepare readme dist directory
+      file:
+        path: "{{ item }}"
+        state: directory
+      with_items:
+        - "{{ readme_output_dir_tmp }}"
+    - name: Generate roles
+      include_tasks: "{{ repo_dir }}/scripts/ansible/generate-roles.yaml"
+      with_items:
+        - { role_type: ClusterRole, prefix: zz- }
```

Comment:
because then you can link it from each namespace and you don't have to create role in each namespace. These cluster roles will not be part of manifests anymore. They will be only part of readme to help users if they will require multinamespace usecase. User can then select if they want to use role or clusterrole.